source/lib/projectz.js

```diff
@@ -386,8 +386,9 @@ class Projectz {
 
 		// Extract repository information
 		/* eslint no-magic-numbers: 0 */
-		let repo = this.mergedPackageData.repository.url || null
-		const githubMatch = (this.mergedPackageData.repository.url || this.mergedPackageData.homepage).match(/github\.com\/(.+?)(?:\.git|\/)?$/)
+		let repo = this.mergedPackageData.repository || null
+		const githubMatch = (this.mergedPackageData.repository.url || this.mergedPackageData.homepage).match(/github\.com\/(.+?)(?:\.git|\/)?$/) ||
+			(this.mergedPackageData.repository.match(/^(.+\/.+)$/) && !this.mergedPackageData.repository.match(/^(.+:.+\/.+)$/))
```

Comment:
It looks like that doesn't work. Basically, all it has to do is to ignore repository names which contain `:` (e.g. `bitbucket:user/repo`). I've checked it, creating GitHub reposities with `:` in it is impossible, they're automatically converted to `-`, so the regex should work fine on that.